ReactWindows/ReactNative.Shared/UIManager/UIManagerModule.cs

```diff
@@ -5,8 +5,12 @@
 using ReactNative.UIManager.Events;
 using System;
 using System.Collections.Generic;
+using System.Windows;
```

Comment:
Or rather, VS was lying to me... it was being used. I'll just move it into the if/def to be sure.